src/net/sourceforge/kolmafia/request/ResearchBenchRequest.java

```diff
@@ -179,6 +198,37 @@ public ResearchBenchRequest(final String research) {
     this.addFormField("r", rfield);
   }
 
+  public static void visitChoice(final String text) {
+    // calculate stomach
+    int wereStomach = 0;
+    if (!text.contains("Osteocalcin injection (10 rp)")
+        && !text.contains("Somatostatin catalyst (20 rp)")
+        && !text.contains("Endothelin suspension (30 rp)")
+        && !text.contains("Synthetic prostaglandin (20 rp)")
+        && !text.contains("Leukotriene elixir (30 rp)")
+        && !text.contains("Thromboxane inhibitor (40 rp)")) {
+      wereStomach += !text.contains("Triiodothyronine accelerator (40 rp)") ? 1 : 0;
+      wereStomach += wereStomach == 1 && !text.contains("Thyroxine supplements (50 rp)") ? 1 : 0;
+      wereStomach +=
+          wereStomach == 2 && !text.contains("Amyloid polypeptide mixture (60 rp)") ? 1 : 0;
+    }
+    Preferences.setInteger("wereProfessorStomach", wereStomach);
+
+    // calculate liver
+    int wereLiver = 0;
+    if (!text.contains("Dopamine slurry (10 rp)")
+        && !text.contains("Relaxin Balm (20 rp)")
```

Comment:
Should be Relaxin balm, or is in my page.  This is what is causing Liver to think it's 13 not 4.

Reply:
Thanks. I saw your PR and duplicated the change here. :)